docs/security.md

```diff
@@ -258,3 +258,32 @@ If you want to use the command `curl`, you can follow the commands:
 openssl x509 -inform der -in $JAVA_HOME/localhost.crt -out certificate.pem
 curl -v -X GET --cacert ./certificate.pem -H "Accept: application/vnd.gravitino.v1+json" -H "Content-Type: application/json" https://localhost:8433/api/version
 ```
+## Cross origin resource filter
+
+### Server configuration
+
+| Configuration item                                | Description                                                                                                                                                                                                                                   | Default value                                 | Required | Since version |
+|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|----------|---------------|
+| `gravitino.server.webserver.enableCorsFilter`     | Enable cross origin resource share filter.                                                                                                                                                                                                    | false                                         | No       | 0.4.0         |
+| `gravitino.server.webserver.allowedOrigins`       | A comma separated list of origins that are allowed to access the resources. Default value is *, meaning all origins.                                                                                                                          | `*`                                           | No       | 0.4.0         |
+| `gravitino.server.webserver.allowedTimingOrigins` | A comma separated list of origins that are allowed to time the resource. Default value is the empty string, meaning no origins.                                                                                                               | ``                                            | No       | 0.4.0         |
```

Comment:
What' the meaning of "time the resource"?

Reply:
This should a term of HTTP. `time the resource` should mean that get metrics time from the server.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Timing-Allow-Origin